content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,127 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+feature:
+  title: Topology Aware Hints
+  description: >
+    Enables topology aware routing with topology hints on EndpointSlices.
+
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in EndpointSlices. This approach tries to keep traffic close to where
+it originated from. This can result in decreased costs and improved performance.
+
+<!-- body -->
+
+## Introduction
+
+Kubernetes clusters are increasingly deployed in multi-zone environments.
+_Topology Aware Hints_ adds new functionality that can help keep traffic within
+the zone it originated from. This concept is commonly referred to as "Topology
+Aware Routing". The "Hints" here refer to a key implementation detail. The
+EndpointSlice controller adds hints to EndpointSlices to allocate each endpoint
```

Comment:
Are the hints labels? Annotations? Something else? Let's mention that here.

Reply:
Updated, but rather messy to get into the same sentence, feedback/suggestions appreciated here. This specifically populates `endpointSlice.endpoints[i].hints.zones[0].name`.

Reply:
One thing to point out then is that the update is to `.endpoints.hints` and not to `.status.*` - right?

Reply:
Updated to say "on each endpoint" instead of "for each endpoint", hopefully that's clearer.